apps/labs/posts/libsf-error-state.md

```diff
@@ -0,0 +1,744 @@
+`install_rpath: '$ORIGIN'` to each extension module. `'$ORIGIN'` means to search in the same folder as the extension
+module.
+
+### Building Wheels on Windows
+
+After setting `install_dir` and `install_rpath` correctly, all but one of SciPy's CI jobs were passing. The sole failing
+job involved building a wheel on Windows. A
+wheel can be thought of as a
+precompiled binary for a Python package. The underlying issue was that Windows does not have support for something like
+`RPATH`, following a less configurable set of rules for determining the search path for shared libraries [].
+
+It took us a day of work to get to this point. Since things were not as straightforward as expected; I took it from here.
+
+At the time I didn't really have any experience developing for Windows and didn't even have a Windows machine available
+at home to use []. I looked up how to run Windows in a VM and got to work.
+
+I found a solution using delvewheel, a tool for bundling shared libraries into wheels on Windows []. It worked on my
+machine but wasn't viable for production []. Fortunately, Ralf had seen this problem before and had a ready made solution:
+manually loading the shared library from within `scipy/special/__init__.py` so it would be available when needed:
+
+```python
+def _load_libsf_error_state():
+    """Load libsf_error_state.dll shared library on Windows
+    libsf_error_state manages shared state used by
+    ``scipy.special.seterr`` and ``scipy.special.geterr`` so that these
+    can work consistently between special functions provided by different
+    extension modules. This shared library is installed in scipy/special
+    alongside this __init__.py file. Due to lack of rpath support, Windows
+    cannot find shared libraries installed within wheels. To circumvent this,
+    we pre-load ``lib_sf_error_state.dll`` when on Windows.
+    The logic for this function was borrowed from the function ``make_init``
+    in `scipy/tools/openblas_support.py`:
+    https://github.com/scipy/scipy/blob/bb92c8014e21052e7dde67a76b28214dd1dcb94a/tools/openblas_support.py#L239-L274
+    """  # noqa: E501
+    if os.name == "nt":
+        try:
+            from ctypes import WinDLL
+            basedir = os.path.dirname(__file__)
+        except:  # noqa: E722
+            pass
+        else:
+            dll_path = os.path.join(basedir, "libsf_error_state.dll")
+            if os.path.exists(dll_path):
+                WinDLL(dll_path)
+
+
+_load_libsf_error_state()
+```
+
+One week in, all CI jobs were passing and the [PR](https://github.com/scipy/scipy/pull/20321) creating
+`libsf_error_state` was merged. We did it! We fixed the bug we'd introduced in `special.errstate` with months to go
+before the next SciPy release—or so we thought.
+
+### Breaking SciPy for MSVC builds
+
+On May 30th, SciPy maintainer and Conda-Forge core member Axel Obermeir (h-vetinari) posted a comment on his open PR,
```

Comment:
I'd not use Axel's name but only his GitHub handle, since he doesn't add his name on purpose to his GitHub profile.